feature-libs/cart/saved-cart/components/details/saved-cart-detail-overview/saved-cart-detail-overview.component.html

```diff
@@ -20,11 +21,13 @@
       </div>
       <div class="cx-summary-card">
         <cx-card [content]="getDateSaved(cart?.saveTime) | async"></cx-card>
-        <cx-card [content]="getCartItems(cart?.totalItems) | async"></cx-card>
+        <cx-card
+          [content]="getCartItems(cart?.totalUnitCount) | async"
+        ></cx-card>
       </div>
       <div class="cx-summary-card">
         <cx-card
-          [content]="getCartQuantity(cart?.totalUnitCount) | async"
+          [content]="getCartQuantity(cart?.totalItems) | async"
```

Comment:
It is probably wrong. quantity is not total items value.

Reply:
I'm looking at the spec, and there's one that uses total items value instead of totalunitcount on zeplin.

I'll change it to total items instead for quantity then